src/test/java/com/thealgorithms/dynamicprogramming/UniquePathsTests.java

```diff
@@ -0,0 +1,62 @@
+package com.thealgorithms.dynamicprogramming;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class UniquePathsTests {
+
+    @Test
+    public void testUniquePaths_3x3() {
+        int result = UniquePaths.uniquePaths(3, 3);
+        assertEquals(6, result);
```

Comment:
```suggestion
        assertEquals(6, UniquePaths.uniquePaths(3, 3));
```

I suggest a similar change in other tests.